src/commands/moderation/warn/deleteWarn.ts

```diff
@@ -49,7 +43,10 @@ const deleteEmbed = makeEmbed({
 
 export const deleteWarn: CommandDefinition = {
     name: ['deletewarn', 'delwarn', 'deletewarning'],
-    requiredPermissions: ['BanMembers'],
+    requirements: {
+        permissions: ['BanMembers'],
```

Comment:
```suggestion
```
Not a comment on the PR, but @benw202 : I'd suggest removing the permissions, it really doesn't have much of a point with the roles there anyway.